package.json

```diff
@@ -34,14 +34,18 @@
   "author": "Lester(Dishu) Lyu",
   "license": "MIT",
   "dependencies": {
+    "async": "^3.2.3",
     "bahttext": "^1.1.0",
     "bessel": "^1.0.2",
+    "cat": "^0.2.0",
     "chai": "^4.2.0",
     "chevrotain": "^7.0.1",
+    "colors": "^1.4.0",
     "coveralls": "^3.1.0",
     "docdash": "^1.2.0",
     "format": "^0.2.2",
     "gh-pages": "^3.1.0",
+    "jq": "^1.7.2",
```

Comment:
Why are we adding jq?

Reply:
Huh weird I dont remember adding it Ill remove them now